src/background/reporting/webrequest-reporter.js

```diff
@@ -48,7 +48,7 @@ if (__PLATFORM__ === 'chromium' || __PLATFORM__ === 'firefox') {
       const hostname = state.tabUrlParts.hostname;
       return (
         !options.blockTrackers ||
-        isPaused(options, hostname) ||
+        !!isPaused(options, hostname) ||
```

Comment:
why is this needed ? `isPaused` should always return a boolean

Reply:
ok, got it. How about we rename `isPaused` to `getPauseDetails`?

Reply:
Ok, it might be the best compromise.